project-base/assets/js/styleguide/styleguide.js

```diff
@@ -1,16 +1,13 @@
+import 'codemirror/addon/scroll/simplescrollbars';
 import CodeMirror from 'codemirror';
 /**
  * Script used for the Styleguide structure only
  */
 
-// eslint-disable-next-line no-global-assign
```

Comment:
Can you please add the upgrade notes (or merge into webpack related ones) as this change is in project-base?
Does it make sense?

Reply:
Ou, sorry, I thought that styleguide was added in version 9. Sorry - 84ea54a40fe041ecfe15868d56ad62bc3c892cf7